### DISCUSSION TITLE
Failed to fetch medium posts

### DISCUSSION THREAD

**Discussion by antra0497** [gautamkrishnar/blog-post-workflow#169] [General]
Hi,

I am trying to fetch my medium post using a cron job, but it is failing all the time, similar is the case with WordPress blog.
Wanted to see if there is a workaround for this or if anyone has successfully done it.

![image](https://user-images.githubusercontent.com/25953832/190656249-6c3c6499-ae22-4415-8786-0ab834f307d3.png)

**Comment by gautamkrishnar** [comment]
Answered on https://github.com/gautamkrishnar/blog-post-workflow/issues/170#issuecomment-1249800009
